cloud-config/oam-server-api.yml

```diff
@@ -4,6 +4,21 @@ coreos:
   update:
     reboot-strategy: off
   units:
+    - name: cfn-signal.service
+      command: start
+      content: |
+        [Unit]
+        Description=CloudFormation signaler
+        After=docker.service
+
+        [Service]
+        ExecStart=/usr/bin/docker run \
```

Comment:
Yes.  Is this syntax right?  (I'm more familiar with `upstart`.)

[Unit]
Description=CloudFormation signaler
After=docker.service

[Service]
Type=oneshot
ExecStart=/usr/bin/docker run \
  --rm \
  quay.io/mojodna/awscli \
  cfn-signal \
  --stack {aws_stack_name} \
  --resource WebServerInstance \
  --region {aws_region}
```